test/classes/Collector.spec.js

```diff
@@ -0,0 +1,126 @@
+import assert from 'power-assert';
+import mockgoose from 'mockgoose';
+import _ from 'lodash';
+import AutherModel from 'models/Author';
+import Collector from 'classes/Collector';
+const AuthorCollector = Collector('author');
+const BookCollector = Collector('book');
+
+describe('/classes/Collector', ()=> {
```

Comment:
__[jshint]__
*  'arrow function syntax (=>)' is only available in ES6 (use esnext option).